helpers/ci_constants.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import json
+from ci.api_lib.helpers.api import OVSClient
+
+
+CONFIG_LOC = "/opt/OpenvStorage/ci/config/setup.json"
+TEST_SCENARIO_LOC = "/opt/OpenvStorage/ci/scenarios/"
+SETTINGS_LOC = "/opt/OpenvStorage/ci/config/settings.json"
+TESTTRAIL_LOC = "/opt/OpenvStorage/ci/config/testtrail.json"
+
+
+class CIConstants(object):
+    """
+    Collection of multiple constants and constant related instances
+    """
+
+    FIO_BIN = {'url': 'http://www.include.gr/fio.bin.latest', 'location': '/tmp/fio.bin.latest'}
```

Comment:
Half of these properties are not used by the api lib ciconstants
Most of these are constants for the integration test one so remove the ones that are not required